### PR TITLE
[14.0][IMP] Switch to OCA copier

### PR DESCRIPTION
**Pull request by rousseldenis**
As the shopinvader copier is quite old and harsh to maintain, I propose to switch to OCA one.

### CODE REVIEW

**Comment by rousseldenis** [comment]
@shopinvader/shopinvader-maintainers

File: .github/workflows/test.yml
Comment (sbidoul) [inline .github/workflows/test.yml]: We need to preserve this.

File: .github/workflows/test.yml
Comment (sbidoul) [inline .github/workflows/test.yml:43]: And probably this too until all dev status issues are fixed.

**Review by sbidoul — CHANGES_REQUESTED** [review]
We need to add new copier questions in the template 
- to make the dev status and license check optional
- to ask for environment variables

File: .github/workflows/test.yml
Comment (rousseldenis) [reply .github/workflows/test.yml]: Could we do this at the same time of switching ?

**Comment by rousseldenis** [comment]
@sbidoul What's your advice for test-requirements.txt url problem ?

I pinged there but ... https://github.com/shopinvader/locomotivecms-python-client/issues/5

File: .github/workflows/test.yml
Comment (rousseldenis) [reply .github/workflows/test.yml:43]: I forced module dev statuses here. If you think ignoring option is better...

**Comment by rousseldenis** [comment]
> We need to add new copier questions in the template
> 
> * to make the dev status and license check optional
> * to ask for environment variables

Ooooh, ok, on OCA side.

**Comment by sbidoul** [comment]
BTW, I'm sure we should downgrade the dev statuses (because it could create issues in downstream projects that check them). I'd rather keep the check silent. We'll fix all that in 16.

**Comment by hparfr** [comment]
What is the status of this PR ?

**Comment by sbidoul** [comment]
> What is the status of this PR ?

Blocked on https://github.com/OCA/oca-addons-repo-template/pull/146

**Comment by simahawk** [comment]
@rousseldenis @sbidoul can we unlock this?
The work on the OCA template has been done https://github.com/OCA/oca-addons-repo-template/pull/179
There are quite some pending PRs on 14.0 that would benefit from this.
Also, the 14.0 branch is red ATM.

**Comment by simahawk** [comment]
FYI I'm testing an updated version here https://github.com/shopinvader/odoo-shopinvader/pull/1352

**Comment by simahawk** [comment]
Build is green here https://github.com/shopinvader/odoo-shopinvader/pull/1352 I'd close this if you don't mind.

**Pull request closed without merging**
